bot.py

```diff
@@ -303,12 +303,12 @@ def select_game(bot, update):
     def selected(bot):
```

Comment:
Line 302 - ``TelegramDeprecationWarning: The @run_async decorator is deprecated. Use the `run_async` parameter of your Handler or `Dispatcher.run_async` instead``

Line 321 - Missing "context" in ``selected(bot)``